src/test/java/htsjdk/samtools/util/IoUtilTest.java

```diff
@@ -46,17 +48,18 @@
 import java.io.OutputStream;
 import java.io.OutputStreamWriter;
 import java.io.StringWriter;
-import java.nio.file.Files;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
 public class IoUtilTest extends HtsjdkTest {
 
-    private static final File SLURP_TEST_FILE = new File("src/test/resources/htsjdk/samtools/io/slurptest.txt");
-    private static final File EMPTY_FILE = new File("src/test/resources/htsjdk/samtools/io/empty.txt");
-    private static final File FIVE_SPACES_THEN_A_NEWLINE_THEN_FIVE_SPACES_FILE = new File("src/test/resources/htsjdk/samtools/io/5newline5.txt");
+
+    private static final File TEST_DATA_DIR = new File ("src/test/resources/htsjdk/samtools/io/");

```

Comment:
Would be better to use `Path` here if possible.

Reply:
OK...though it makes things a bit more cumbersome

Reply:
Oh. nice. thanks.